### ISSUE TITLE
Autofill provider information from PECOS

### ISSUE THREAD

**Issue by cecilia-donnelly** [OpenTechStrategies/psm#378]
@jvasile points out that when a provider enters their NPI, we can query PECOS for their address and other information, and autofill the form with that.

**Comment by kfogel** [comment]
Yes, see issue #273 (which might be a dup of this one, but I'm not sure yet).  I've linked from there to what might be a spec for the format of the information coming from PECOS.
